src/pupil_labs/neon_recording/stream/av_stream/video_stream.py

```diff
@@ -33,6 +35,15 @@ def height(self) -> int | None:
         """Height of image in stream"""
         return self.av_reader.height
 
+    def sample(
```

Comment:
This is just a proof of concept implementation on of the SampledStream class underneath. The proper implementation would probably to remove the SampledStream and instead access an AVFrameArray class

Reply:
Why can't this be put on SampledStream instead, that way it applies to all streams that inherit it?

Right now this proposal involves adding a .sample() to each and every stream, gaze/imu/scene etc.